patterns/hidden-404.php

```diff
@@ -7,15 +7,15 @@
 ?>
 
 <!-- wp:heading {"style":{"typography":{"fontSize":"clamp(4rem, 40vw, 20rem)","fontWeight":"400","lineHeight":"1.25"}},"className":"has-text-align-center"} -->
-<h2 class="has-text-align-center" style="font-size:clamp(4rem, 40vw, 20rem);font-weight:400;line-height:1.25"><?php echo esc_html( _x( '404', 'Error code for a webpage that is not found.', 'twentytwentythree' ) ); ?></h2>
+<h2 class="has-text-align-center" style="font-size:clamp(4rem, 40vw, 20rem);font-weight:400;line-height:1.25"><?php echo esc_html_x( '404', 'Error code for a webpage that is not found.', 'twentytwentythree' ); ?></h2>
 <!-- /wp:heading -->
 
 <!-- wp:paragraph {"align":"center","fontSize":"small"} -->
-<p class="has-text-align-center"><?php esc_html_e( 'This page could not be found.', 'twentytwentythree' ); ?></p>
+<p class="has-text-align-center"><?php echo esc_html_x( 'This page could not be found.', 'Message to convey that a webpage could not be found', 'twentytwentythree' ); ?></p>
 <!-- /wp:paragraph -->
 
 <!-- wp:spacer {"height":"var(--wp--preset--spacing--100)"} -->
 <div style="height:var(--wp--preset--spacing--100)" aria-hidden="true" class="wp-block-spacer"></div>
 <!-- /wp:spacer -->
 
-<!-- wp:search {"label":"<?php echo esc_html( _x( 'Search', 'label', 'twentytwentythree' ) ); ?>","placeholder":"<?php echo esc_html( _x( 'Search...', 'placeholder', 'twentytwentythree' ) ); ?>","showLabel":false,"width":70,"widthUnit":"%","buttonText":"<?php esc_html_e( 'Search', 'twentytwentythree' ); ?>","buttonUseIcon":true,"align":"center"} /-->
+<!-- wp:search {"label":"<?php echo esc_html_x( 'Search', 'label', 'twentytwentythree' ); ?>","placeholder":"<?php echo esc_html_x( 'Search...', 'placeholder for search field', 'twentytwentythree' ); ?>","showLabel":false,"width":70,"widthUnit":"%","buttonText":"<?php esc_html_e( 'Search', 'twentytwentythree' ); ?>","buttonUseIcon":true,"align":"center"} /-->
```

Comment:
Should the `buttonText` be wrapped in the `esc_html_x()` function as well? It's currently using `esc_html_e()`.

Reply:
The button text does not need the additional context for translators. Therefore the `esc_html_e()` is sufficient.

Reply:
Ah I see, great ok!